lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -0,0 +1,169 @@
+.container {
+    background-color: $color-bg-white;
+    display: inline-flex;
+    flex-direction: column;
+
+    button,
+    a {
+        background: $color-bg-white;
+    }
+
+    svg {
+        color: $color-text-blue-dark;
+    }
+
+    button:focus,
+    button:hover,
+    a:focus,
+    a:hover {
+        background: $color-bg-gray-light;
+        color: $color-link-black;
+    }
```

Comment:
This shouldn't be needed if we used <Button> in the template, and is causing the "Save" button to be difficult to see when editing any field

Reply:
That simplified much of the styling. Thank you.